tests/Meilisearch.Tests/DocumentTests.cs

```diff
@@ -134,10 +135,8 @@ public async Task BasicDocumentsAdditionInBatches()
             // Add the documents
             Movie[] movies =
             {
-                new Movie { Id = "1", Name = "Batman" },
-                new Movie { Id = "2", Name = "Reservoir Dogs" },
-                new Movie { Id = "3", Name = "Taxi Driver" },
-                new Movie { Id = "4", Name = "Interstellar" },
+                new Movie { Id = "1", Name = "Batman" }, new Movie { Id = "2", Name = "Reservoir Dogs" },
+                new Movie { Id = "3", Name = "Taxi Driver" }, new Movie { Id = "4", Name = "Interstellar" },
```

Comment:
Is it because of the linter because I find it less legible on several lines? Especially for tests

Reply:
Nope, this was because I am using a custom formatter that sets line limits, and this was just it reformatting on saving the document.
apologies about that !